torch/_inductor/codegen/common.py

```diff
@@ -289,28 +289,6 @@ def all_in_parens(string):
             return string
         return f"({string})"
 
-    def _print_Pow(self, expr):
```

Comment:
I moved this down to `PythonPrinter` since `self._helper_sqrt` is only defined there. And then I inlined `_helper_sqrt` because it didn't really seem necessary to have it as a separate method

(`CppPrinter` has its own `_print_Pow` implementation)